Mathlib/Algebra/Polynomial/Expand.lean

```diff
@@ -5,6 +5,7 @@ Authors: Kenny Lau
 -/
 import Mathlib.Algebra.CharP.Lemmas
 import Mathlib.Algebra.Polynomial.Derivative
+import Mathlib.Algebra.Polynomial.FieldDivision
```

Comment:
This increases the imports to quite a few files. Do you think you could reorder things around so as to remove this import increase?

Reply:
Added! #20268

Reply:
Maybe I didn't get your question - reorder which things?

Reply:
Move lemmas across files is what I mean

Reply:
I have to admit I also don't follow what the 'import increase' means exactly.

Right now the only new import line is line 8 'import Mathlib.Algebra.Polynomial.FieldDivision'.
Are you suggesting that we can move the lemmas in `Mathlib.Algebra.Polynomial.FieldDivision` to other existing imports, and delete line 8?

Reply:
I don't actually know what the answer is here, but I expect it to be one of:
* Move the new lemmas to another existing file
* Create a new file for the new lemmas